.drone.yml

```diff
@@ -1853,7 +1853,7 @@ steps:
   image: docker
   pull: if-not-exists
   commands:
-  - apk add --no-cache make aws-cli
+  - apk add --no-cache make aws-cli go
   - chown -R $UID:$GID /go
```

Comment:
`apk add go` installs `go` to `/usr/lib/go`, so the `chown` on `/go` isn't doing to do much..

Reply:
`/go` is the root for building (for example, `/go/src/github.com/gravitational/teleport`), so that's why the `chown` is there. It's not actually where `go` is installed. We just need `go` for some commands in `Makefile`. The two commands are unrelated.